### PR TITLE
NoDMA platform validate bug fixes

### PR DESCRIPTION
**Pull request by houlz0507**

### CODE REVIEW

**Comment by houlz0507** [comment]
retest this please

File: src/runtime_src/core/common/api/xrt_bo.cpp [hunk 1]
```diff
@@ -346,15 +346,31 @@ class buffer_import : public bo_impl
   }
 };
 
+// class buffer_dbuf - device only side buffer
+//
+// Kernel driver allocated host side buffer.  The host side buffer
+// is allocated in physical memory by kernel driver.
```
Comment (stsoe) [inline src/runtime_src/core/common/api/xrt_bo.cpp:352]: Please update or remove comment

**Review by stsoe — COMMENTED** [review]
Looks good to me.
Just fix the comment for buffer_dbuf.

File: src/runtime_src/core/common/api/xrt_bo.cpp (hunk 1 above)
Comment (houlz0507) [reply src/runtime_src/core/common/api/xrt_bo.cpp]: removed

**Review by maxzhen — APPROVED** [review]

**Review by xdavidz — APPROVED** [review]

**Review by chienwei-lan — APPROVED** [review]

**Pull request merged**